teuthology/task/internal/__init__.py

```diff
@@ -19,7 +19,7 @@
 from teuthology.exceptions import VersionNotFoundError
 from teuthology.job_status import get_status, set_status
 from teuthology.orchestra import cluster, remote, run
-
+from .redhat import setup_cdn_repo, setup_base_repo, setup_additional_repo, setup_stage_cdn # noqa
```

Comment:
Could you please use full path to the redhat module?

Reply:
And remove # noqa

Reply:
@rakeshgm can you see if the test passes with noqa and full path?

Reply:
@vasukulkarni I would prefer to follow one style in imports.

Reply:
also in some circumstances it can fail

Reply:
Done. relative import added

Reply:
test passes with full path.

Reply:
tests failed because flake8 reported unused imports

Reply:
so why this imports needed if they are not used?

Reply:
they are actually being called from run.py https://github.com/ceph/teuthology/blob/master/teuthology/run.py#L236.

Reply:
> they are actually being called from run.py https://github.com/ceph/teuthology/blob/master/teuthology/run.py#L236.

@rakeshgm the code you're referring is not used for calling them, it is just compose the task list, the modules for them are loaded automatically in other place, see as an example other jobs in the task list are not imported directly here in this module.

Reply:
@kshtsk , I intended to mention the same. sorry I did not phrase it correctly.

Reply:
@kshtsk  so to fix the flake8 errors, I can move the tasks from `redhat` and add them in `__init__` which might fix the problem else we could check for flake8 options to ignore such errors i.e if you think its a good idea. 
Or any other suggestions.

Reply:
@rakeshgm no, just remove this import line, and that's it.

Reply:
`setup_cdn_repo, setup_base_repo, setup_additional_repo, setup_stage_cdn` tasks are present in `redhat.py` unlike others in `__init__` for example in run.py we have somthing like this `{'internal.setup_cdn_repo': None}` .
importing this line gets the task.

Reply:
Ah, I see, why not using `{'internal.redhat.setup_cdn_repo': None}` then?

Reply:
I tried that a while ago. the split here fails. 
https://github.com/ceph/teuthology/blob/master/teuthology/run_tasks.py#L21

Reply:
I pushed a small [change here](6261eae46675eacc515f00bc4f3f1b070dbeb85b)

Reply:
I pushed a small change [here](https://github.com/ceph/teuthology/pull/1479/commits/6261eae46675eacc515f00bc4f3f1b070dbeb85b) to this PR.

Reply:
looks like the tests passed.

Reply:
> I tried that a while ago. the split here fails.
> https://github.com/ceph/teuthology/blob/master/teuthology/run_tasks.py#L21
> returns more than 2.

Oh... shoot, it does not support submodules then... Seems like redhat is the first ever module is trying to do so.
I guess we could add a support for submodules.

Reply:
> I pushed a small change [here](https://github.com/ceph/teuthology/pull/1479/commits/6261eae46675eacc515f00bc4f3f1b070dbeb85b) to this PR.

That's too hacky, I would go with `# noqa` tag accompanying a comment why it is needed?

Reply:
however... it's fine....

Reply:
we can add support for submodules later...

Reply:
yeah the split and support for single module was the reason I think we did that, I suggest we keep the original code and add comment and move on until the support exists in teuthology.

Reply:
reverted, add full path import along with `noqa`